### DISCUSSION TITLE
[운영체제 8일차] 메인 메모리

### DISCUSSION THREAD

**Discussion by seongeun42** [Shortudy-10th/tech-for-developer#90] [오늘의 주제]
메인 메모리(물리) 질문 2인조 1개, 3인조 2개

**Comment by LireEruel** [comment]
메인 메모리란 무엇이고, 왜 중요한가요?

**Comment by Jihn0118** [comment]
단편화가 무엇이고, 단편화의 종류는 어떤 것들이 있나요?

**Comment by seongeun42** [comment]
메모리의 절대 주소와 상대 주소에 대해 말해주세요.

**Comment by seongeun42** [comment]
메모리 배치 정책에 대해 설명해주세요.

**Comment by aeroej** [comment]
캐시 메모리는 무엇이고, CPU와 메인 메모리 사이에서 어떤 역할을 하나요?
